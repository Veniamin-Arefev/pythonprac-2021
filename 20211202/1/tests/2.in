class C(metaclass=dump):
    def __init__(self, string):
        self.val = string

    def __add__(self, other):
        return (self.val + other.val)

    def __repr__(self):
        return f"{self.val}"

a, b = C("geg"), C("kek")
print(a + b)