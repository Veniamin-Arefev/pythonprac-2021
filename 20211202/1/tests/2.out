__init__: ('geg',), {}
__init__: ('kek',), {}
__repr__: (), {}
__add__: (kek,), {}
gegkek
